actors/builtin/multisig/multisig_test.go

```diff
@@ -31,15 +31,15 @@ func TestConstruction(t *testing.T) {
 
 	receiver := tutil.NewIDAddr(t, 100)
 	anne := tutil.NewIDAddr(t, 101)
+	anneNonId := tutil.NewBLSAddr(t, 1)
 
 	bob := tutil.NewIDAddr(t, 102)
 	bobNonId := tutil.NewBLSAddr(t, 2)
 
 	charlie := tutil.NewIDAddr(t, 103)
 
-	builder := mock.NewBuilder(context.Background(), receiver).WithCaller(builtin.InitActorAddr, builtin.InitActorCodeID)
-
 	t.Run("simple construction", func(t *testing.T) {
+		builder := mock.NewBuilder(context.Background(), receiver).WithCaller(builtin.InitActorAddr, builtin.InitActorCodeID)
```

Comment:
Why did you move and duplicate this line in all the cases? They all need it, and if something needs something special, you can shadow it or use another name.

Reply:
Apologies, this is fixed.